Project.toml

```
@@ -1,6 +1,6 @@
 name = "ChainRulesCore"
 uuid = "d360d2e6-b24c-11e9-a2a3-2a2ae2dbcce4"
-version = "0.2.0"
+version = "0.3.0-DEV"
```

```suggestion
version = "0.2.1-DEV"
```

Turns out this is nonbreaking.
or at least it doesn't break ChainRules.jl
so I am happy to call it a nonbreaking release.
Particularly while it is still marked `--DEV`.
I guess it is just making `extern` do what it says it does in the docs.